build/nuget/uno.winui.common.targets

```diff
@@ -56,7 +56,7 @@
 			<DefineConstants>$(DefineConstants);UNO_HAS_NO_IDEPENDENCYOBJECT</DefineConstants>
 		</PropertyGroup>
 
-		<PropertyGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETStandard' or '$(TargetFrameworkIdentifier)'=='.NETCoreApp'">
+		<PropertyGroup Condition="'$(UnoRuntimeIdentifier)'=='Skia' or '$(UnoRuntimeIdentifier)'=='WebAssembly' or '$(UnoRuntimeIdentifier)'=='Reference'">
```

Comment:
This might not work properly :/

UnoRuntimeIdentifier is never set to Reference in applications it seems. This needs a bit more work.